config/federation.js

```diff
@@ -18,6 +18,7 @@ const config = {
     ...deps,
     react: { singleton: true, requiredVersion: deps.react },
     "react-dom": { singleton: true, requiredVersion: deps["react-dom"] },
+    marked: { eager: true },
```

Comment:
The preview page doesn't actually reference this (I don't know why, but the preview React component runs within the main document scope, not the preview iframe).

I originally added this because `marked` wouldn't be visible from React when injected into the preview iframe, and when I required it the same way as React, webpack complained that I needed this option enabled. In retrospect, I just needed to put the script tag in `static/admin/index.html`, and it works as expected.

Reply:
OK. Either way, you're going to need to do the bootstrap loading script in order to use `<Shell />` or any other components from host in the previews